<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2//EN">
<HTML>
 <HEAD>
   <TITLE> [mod_python] compile problem
   </TITLE>
   <LINK REL="Index" HREF="index.html" >
   <LINK REL="made" HREF="mailto:mod_python%40modpython.org?Subject=%5Bmod_python%5D%20compile%20problem&In-Reply-To=">
   <META NAME="robots" CONTENT="index,nofollow">
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="007827.html">
   <LINK REL="Next"  HREF="007829.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
<table border=0 width="100%"><tr><td valign="top">
   <H1>[mod_python] compile problem</H1>
    <B>Ken Kinder</B> 
    <A HREF="mailto:mod_python%40modpython.org?Subject=%5Bmod_python%5D%20compile%20problem&In-Reply-To="
       TITLE="[mod_python] compile problem">ken at kenkinder.com
       </A><BR>
    <I>Sun Nov 26 01:17:26 EST 2000</I>
    <P><UL>
        <LI>Previous message: <A HREF="007827.html">[mod_python] please help...
</A></li>
        <LI>Next message: <A HREF="007829.html">[mod_python] runtime error
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#7828">[ date ]</a>
              <a href="thread.html#7828">[ thread ]</a>
              <a href="subject.html#7828">[ subject ]</a>
              <a href="author.html#7828">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>The odd thing is, on my Red Hat 7 Linux, this works fine, but on a Debian
2.1 box, it doesn't work. Here's what happens:

The error happened on make. Here's my configure command:

./configure --with-apache=../apache_1.3.14 --with-apxs=/usr/local/apache/bin/apxs

and make was make dso:

In file included from /usr/local/src/apache_1.3.14/src/include/ap_config.h:76,
                 from /usr/local/src/apache_1.3.14/src/include/httpd.h:71,
                 from /usr/local/src/mod_python-2.6.3/src/include/mod_python.h:85,
                 from mod_python.c:61:
/usr/local/src/apache_1.3.14/src/include/ap_config_auto.h:6: unterminated `#if' conditional
make[1]: *** [mod_python.o] Error 1
make[1]: Leaving directory `/usr/local/src/mod_python-2.6.3/src'
make: *** [do_dso] Error 2


-- 
Ken Kinder
www.kenkinder.com

</PRE>
<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="007827.html">[mod_python] please help...
</A></li>
	<LI>Next message: <A HREF="007829.html">[mod_python] runtime error
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#7828">[ date ]</a>
              <a href="thread.html#7828">[ thread ]</a>
              <a href="subject.html#7828">[ subject ]</a>
              <a href="author.html#7828">[ author ]</a>
         </LI>
       </UL>

</td>
<td align="right" valign="top">
<script type="text/javascript"><!--
google_ad_client = "pub-9718360309690383";
google_ad_width = 120;
google_ad_height = 600;
google_ad_format = "120x600_as";
google_color_border = "CCCCCC";
google_color_bg = "FFFFFF";
google_color_link = "000000";
google_color_url = "666666";
google_color_text = "333333";
//--></script>
<script type="text/javascript"
  src="DISABLEhttp://pagead2.googlesyndication.com/pagead/show_ads.js">
</script>
</td>
</table>

<hr>
<a href="http://mailman.modpython.org/mailman/listinfo/mod_python">More information about the Mod_python
mailing list</a><br>
</body></html>
